playwright/package.json

```diff
@@ -3,6 +3,7 @@
   "license": "MIT",
   "scripts": {
     "test": "playwright test",
+    "test:desktop": "playwright test",
```

Comment:
This is so that `yarn test:desktop` does the same thing as what `yarn test` was doing in CI.

The fact our playwright tests are desktop by default is an interesting choice.